app/controllers/authentications_controller.rb

```diff
@@ -35,7 +35,12 @@ def destroy
         # Bryan: TESTED
         flash[:alert] = 'Bad idea!'
       elsif @authentication.destroy
-        flash[:notice] = 'Successfully removed profile.'
+	user = User.find(current_user.id)
+	if user.update_attributes(github_profile_url: nil)
+          flash[:notice] = 'Successfully removed profile.'
+	else
+          flash[:notice] = 'Github profile url could not be removed.'
```

Comment:
@touchps following up on this - was this the flash part that wasn't reached by the tests?  Do we even know what might cause a failure here?